crates/compiler/src/pass/analysis/module_map.rs

```diff
@@ -289,11 +316,30 @@ pub struct ModuleMap {
 
 impl ModuleMap {
     /// Creates a new instance of the output data for the module mapping pass.
+    ///
+    /// # Anonymous Modules
+    ///
+    /// If the module is anonymous—in other words that its name is an empty
+    /// string—it will have a name generated at random. Please note that the
+    /// underlying RNG **cannot be relied upon to be cryptographically secure**,
```

Comment:
Because I have seen more than enough people make stupid assumptions about what they can rely on an RNG for, so I am very clear about it. The below comment is for developers only; clients would mostly read the API doc.